src/components/ui/Section.jsx

```
@@ -0,0 +1,27 @@
+import { twMerge } from "tailwind-merge";
+
+const Section = ({
+  id,
```

ESLint

[error] 'id' is missing in props validation

(react/prop-types)